plugins.js

```diff
@@ -16,6 +17,13 @@ const cache = new Config();
 // modules path
 const path = resolve(homedir(), '.hyperterm_plugins');
 const localPath = resolve(homedir(), '.hyperterm_plugins', 'local');
+const availableExtensions = [
+  'onApp', 'onWindow', 'onUnload', 'middleware',
+  'reduceUI', 'reduceSessions', 'decorateMenu',
+  'decorateTerm', 'decorateHyperTerm', 'decorateTab',
+  'decorateNotification', 'decorateNotifications',
+  'decorateTabs', 'decorateConfig', 'decorateEnv'
+];
```

Comment:
A Set makes sense, but I'm not sure if I understand where you'd call `has` on it? Something like `Object.keys(mod).some(name => availableExtensions.has(name));` or were you thinking of a different place altogether?